docs/proposals/node-agent/README.md

```diff
@@ -0,0 +1,42 @@
+# Node Agent
+
+## Requirements
+
+### Which service must the domain provide
+
+The node agent manages `orka` workloads on a specific node. There is one agent per node.
+
+* A node is a physical or virtual machine in the cluster, that is managed by the `orka` control plane.
+* A workload is the smallest unit of compute deployable in orka. It can be e.g. a container, a virtual machine, etc.
+
+### Scope
```

Comment:
```suggestion
### Scope

The node agent manages `orka` workloads on a specific node. There is one agent per node.

* A node is a physical or virtual machine in the cluster, that is managed by the `orka` control plane.
* A workload is the smallest unit of compute deployable in orka. It can be e.g. a container, a virtual machine, etc.

```

Reply:
Fixed in https://github.com/dev-sys-do/orka/pull/1/commits/4c226c00975c5232c7216a4d699092bc85f346d6